### PR TITLE
Bump version, fix layout, add DiffUtil example

### PR DESCRIPTION
**Pull request by rogerhu**

### CODE REVIEW

File: app/src/main/java/com/demo/recyclerviewdemo/Contact.java [hunk 1]
```diff
@@ -4,14 +4,21 @@
 import java.util.List;
 
 public class Contact {
+    private int mId;
+
```
Comment (ravibhoraskar) [inline app/src/main/java/com/demo/recyclerviewdemo/Contact.java:8]: Nit: Remove newline

File: app/src/main/java/com/demo/recyclerviewdemo/ContactsAdapter.java [hunk 2]
```diff
@@ -13,23 +15,31 @@
...
-        // and does not update automatically.
-        notifyItemRangeInserted(insertionPosition, newContacts.size());
+        submitList(mContacts); // DiffUtil takes care of the chekc
     }
 
-    @Override
-    public int getItemCount() {
-        return mContacts.size();
-    }
+    public static final DiffUtil.ItemCallback<Contact> DIFF_CALLBACK =
```
Comment (ravibhoraskar) [inline app/src/main/java/com/demo/recyclerviewdemo/ContactsAdapter.java]: Let's define this above the function definitions, where other member fields are defined.. ?

**Review by ravibhoraskar — COMMENTED** [review]
Missing test plan — does the app build and run correctly with these changes? Screenshots might be useful too.

**Comment by rogerhu** [comment]
It's a demo app.  If you're asking for .travis.yml so Travis runs will pass a build, that's easy enough.  The app actually didn't even work to render the right page given the match_parent issues.

**Comment by nesquena** [comment]
@rogerhu I added you access so you can merge this when ready

**Pull request merged**